src/plugins/intel_cpu/src/nodes/topk.cpp

```diff
@@ -1095,7 +1094,7 @@ struct jit_uni_topk_kernel_f32 : public jit_uni_topk_kernel, public jit_generato
             shl(reg_out, 2);
             break;
         default:
-            assert(!"incorrect data size rate");
+            OPENVINO_DEBUG_ASSERT(false, "incorrect data size rate: rate=", rate);
```

Comment:
```suggestion
            OPENVINO_DEBUG_THROW("incorrect data size rate: rate=", rate);
```
Maybe `OPENVION_THROW` should have also debug version?